dashboard/src/containers/Root.tsx

```diff
@@ -18,9 +18,16 @@ async function initLocale() {
 
 function Root() {
   const [i18nConfig, setI18nConfig] = useState(I18n.getDefaultConfig());
+  // Currently we just support 'dark' and 'light' themes
+  // TODO(agamez): add manual override for this setting
+  const theme =
+    window.matchMedia && window.matchMedia("(prefers-color-scheme: dark)").matches
+      ? "dark"
+      : "light";
   useEffect(() => {
```

Comment:
also, better to use a different `useEffect` to set this. I believe there is no relation between `initLocale` and setting the cds-theme property.